chapters/ja/event/1.mdx

```diff
@@ -0,0 +1,204 @@
+# Chapter2公開記念イベント
```

Comment:
Similar comment as above

Reply:
I fixed in [6075579](https://github.com/huggingface/course/pull/241/commits/60755792eb0ea244cd7f919b41aa011edde851e9)

Thanks!